todo_project/urls.py

```
@@ -1,7 +1,10 @@
 from django.urls import path, include
+from django.views.generic import RedirectView
```

Ruff (0.11.9)

`django.views.generic.RedirectView` imported but unused

Remove unused import: `django.views.generic.RedirectView`

(F401)